### PR TITLE
Group 63 fetch all comments in frontend and show as per post

### PR DESCRIPTION
**Pull request by udkreddy**

### CODE REVIEW

File: Frontend/src/app/feed/feed.component.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { DefaultService } from "../default.service";
 import { environment } from 'src/environments/environment';
 import { ToastrService } from 'ngx-toastr';
 import { DomSanitizer } from '@angular/platform-browser';
+import { AuthenticationService } from '../services/authentication.service';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
```
Comment (harish426) [inline Frontend/src/app/feed/feed.component.ts]: I am getting error that we are unable to import module

File: Frontend/src/app/feed/feed.component.ts [hunk 2]
```diff
@@ -12,7 +14,8 @@ import { DomSanitizer } from '@angular/platform-browser';
 export class FeedComponent {
```
Comment (harish426) [inline Frontend/src/app/feed/feed.component.ts:14]: it reflecting error at this point, check this onece

File: Frontend/src/app/feed/feed.component.ts (hunk 1 above)
Comment (udkreddy) [reply Frontend/src/app/feed/feed.component.ts]: Try executing `npm install --force`

File: Frontend/src/app/feed/feed.component.ts (hunk 2 above)
Comment (udkreddy) [reply Frontend/src/app/feed/feed.component.ts:14]: This is related to the issue mentioned above

**Comment by harish426** [comment]
stil I am encountering 404 error at commenting

**Comment by harish426** [comment]
Scrolling is not working for comments and data is not dynamically visible after comment

**Comment by harish426** [comment]
now it is working well, I am merging to dev

**Review by harish426 — COMMENTED** [review]
approved

**Pull request merged**